eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/configuration/SubmittedVariantAccessioningConfiguration.java

```diff
@@ -64,8 +68,9 @@ public SubmittedVariantAccessioningDatabaseService submittedVariantAccessioningD
     }
 
     @Bean
-    public MonotonicAccessionGenerator<SubmittedVariantModel> submittedVariantAccessionGenerator() {
+    public MonotonicAccessionGenerator<ISubmittedVariant> submittedVariantAccessionGenerator() {
         ApplicationProperties properties = applicationProperties();
+        logger.debug("Using application properties: " + properties.toString());
```

Comment:
These should be reported by Spring Boot anyway, could you please check that they are not the same?

Reply:
couldn't find the values of the properties in the logs, even with boot at "debug" log level.

The only messages I found are similar to 
> Adding PropertySource 'applicationConfig: [classpath:/application.properties]' [...]
> Adding PropertySource 'class path resource [accession-test.properties]' [...]

which just says everything spring found, and 
> Retrieved ApplicationContext [...]  propertySourceLocations = '{classpath:accession-test.properties}' [...]

which just re-states what the annotation `@TestPropertiesSources` has in our code. 

Personally didn't find that information useful to make sure the properties taken were the expected ones. Actually, at first I used `@PropertiesSources` and spring didn't use the expected values.